### ISSUE TITLE
Content updates - bug bash

### ISSUE THREAD

**Issue by stacymcauliffe** [patternfly/patternfly-org#860]
Issue to collect small content edits and recommendations captured during the bug bash.

**Comment by stacymcauliffe** [comment]
“Because building applications that work for everyone, regardless of ability, is just the right thing to do.”  

I think it would be nice to reword this to say “of all abilities” rather than “regardless of ability” - more positive and inclusive wording

**Comment by mcoker** [comment]
On https://www.patternfly.org/v4/get-started/developers Should there be a colon or hyphen after "fork the project" and "install the project"

> Fork the project Fork and clone the PatternFly React Seed project from https://github.com/patternfly/patternfly-react-seed
> Install the project Follow the steps outlined in the React seed README to install.

**Comment by schizari** [comment]
Not sure if this is a bug, but on the homepage, towards the bottom of the page in PF Principles section it says:
Get started for developers
Get started for designers
<img width="736" alt="Screen Shot 2019-04-26 at 11 10 19 AM" src="https://user-images.githubusercontent.com/33766334/56827908-f1fc7700-6814-11e9-955c-be42fb4c2c80.png">


After clicking on the link, it takes you to a page (https://www.patternfly.org/v4/get-started/about) where the terminologies are not consistent with the links. 
<img width="696" alt="Screen Shot 2019-04-26 at 11 11 34 AM" src="https://user-images.githubusercontent.com/33766334/56827915-f88aee80-6814-11e9-8422-feabec8b4486.png">

Would it make sense to reword what we have on the homepage to this:
Develop with PatternFly
Design with PatternFly

**Comment by mcoker** [comment]
Under https://www.patternfly.org/v4/get-started/developers, there is only one command, but we say "run the following commands".

> Using [yarn/npm], run the following commands

**Comment by mcoker** [comment]
On https://www.patternfly.org/v4/get-started/developers Missing a couple of spaces

> If you’re using typescript...See typescript.example.jsfor an example configuration.

**Comment by mcoker** [comment]
On https://www.patternfly.org/v4/get-started/developers, should we have a comma after "Install NPM", or . should it be a hyphen or colon or something different?

> Install NPM, NPM is a package manager that’s bundled with Node.js.

**Comment by mcoker** [comment]
On https://www.patternfly.org/v4/get-started/developers we have both "sass" and "SASS" - we should be consistent. It's actually "Sass" on https://sass-lang.com/

**Comment by sbuenafe-rh** [comment]
On the Getting Started page,  (https://www.patternfly.org/v4/get-started/about) the H1: About Patternfly 4, the 'f' should be capitalized to be "PatternFly 4"

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/get-started/developers shouldn't `@patternfly/patternfly-next` just be `@patternfly/patternfly`?

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/get-started/developers "A minified version of the patternfly.css:" should probably be "A minified version of patternfly.css:"

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/get-started/developers I think references to `<link>` code like `link rel="stylesheet" href="css/patternfly.css"` should be `<link rel="stylesheet" href="css/patternfly.css">`

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/get-started/developers looks kind of odd that 4, 5, and 6 are nested.

<img width="989" alt="Screen Shot 2019-04-26 at 1 32 19 PM" src="https://user-images.githubusercontent.com/35148959/56828828-c46cf900-6827-11e9-8b11-c06933678111.png">

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/get-started/developers references to `utilities/variables.scss ` to update global variables should be `sass-utilities/scss-variables.scss`

**Comment by catrobson** [comment]
on https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/modal, at the bottom there is a comma after Modal in the React Components list
![image](https://user-images.githubusercontent.com/3739227/56829096-d30bde00-6830-11e9-9e2c-7479d676002d.png)

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/design-guidelines/styles/colors, should this just say "link text" instead of "link texts"?

> Make sure that, on hover, link texts provides

**Comment by TheRealJon** [comment]
> on https://www.patternfly.org/v4/get-started/developers looks kind of odd that 4, 5, and 6 are nested.
> 
> <img alt="Screen Shot 2019-04-26 at 1 32 19 PM" width="989" src="https://user-images.githubusercontent.com/35148959/56828828-c46cf900-6827-11e9-8b11-c06933678111.png">

Also, missing spaces on 1 and 2

**Comment by tlabaj** [comment]
"About Patternfly 4" should be "About PatternFly 4" on the About page.

**Comment by tlabaj** [comment]
Sentences case is off in the **Get Started->Develop** section. 
Some of the bold text seems to be a heading(e.g. **Install the project** Follow the steps outlined ) an some seems to be part of the sentence (e.g. **If you’re starting from scratch and would like to use a sample project**, we recommend starting...)

![image](https://user-images.githubusercontent.com/39532947/56829718-81fce980-6832-11e9-8d47-166cc0d0a664.png)

**Comment by mcoker** [comment]
on https://www.patternfly.org/v4/design-guidelines/styles/typography#usage-guidelines we use a mix of "Line height" and "line-height" - should that be consistent?

> Line height
> When laying out text-based content, you need to factor in line-height when measuring spacing. Line height is a relative number used in CSS that represents a ratio to the text size. For example, body text uses a line height of 1.5. Body text size is 16px. 16*1.5 = 24px line height. So, when creating designs in design software, you must include the 24px line height as a part of the text.

**Comment by rachael-phillips** [comment]
Closing this as this work has been completed
